examples/src/test/scala/akka/projection/jdbc/docs/OffsetSchemaSpec.scala

```diff
@@ -0,0 +1,73 @@
+package akka.projection.jdbc.docs
+
+import scala.io.Source
+
+import akka.projection.jdbc.internal.DefaultDialect
+import akka.projection.jdbc.internal.MSSQLServerDialect
+import akka.projection.jdbc.internal.MySQLDialect
+import akka.projection.jdbc.internal.OracleDialect
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+/**
+ *
+ */
+class OffsetSchemaSpec extends AnyWordSpecLike with Matchers {
```

Comment:
This test will assert the SQL in the `create-table-xxx.sql` is equivalent to that in the dialect. There's a `normalize` function to help the String comparison.